docs/gencertchain.md

```diff
@@ -0,0 +1,18 @@
+# gencertchain
+
+A command-line tool for generating Certificate-Chain cbor file.
+
+# Usage
+
+This command generates an Certificate-Chain cbor file.
+
+```sh
```

Comment:
This reason I suggest ShellSession instead of bash or sh is that it does a slightly different syntax highlighting for $ prompts.

Reply:
Oh, I realized, sorry for my misunderstood.
ShellSession provides a bit cooler highlighting, I didn't know. Thank you.